README.md

```diff
@@ -38,22 +38,20 @@ bob.name = "Bob";
 bob.age = 42;
 
 String bobJson = jsonb.toJson(bob);
-System.out.println(bobJson); // {"id":1234,"name":Bob","age":42}
+System.out.println(bobJson); // {"id":1234,"name":"Bob","age":42}
 ```
 
 Likewise, JSON data can be deserialized back into Java objects:
 ```java
 Jsonb jsonb = JsonbBuilder.create();
 
-String aliceJson = "{\"id\":5678,\"name\":Alice\",\"age\":42}";
+String aliceJson = "{\"id\":5678,\"name\":\"Alice\",\"age\":42}";
 User alice = jsonb.fromJson(aliceJson, User.class);
 ```
 
 ## Links
 
 - Official web site: https://eclipse-ee4j.github.io/jsonb-api
-- JSON-B user guide: https://javaee.github.io/jsonb-spec/users-guide.html
```

Comment:
same comment as a previous PR -- need to ensure the JSON-B user guide does not get lost

Reply:
Done it. See https://github.com/eclipse-ee4j/jsonb-api/pull/150